core/spec/lib/spree/bus_spec.rb

```diff
@@ -4,8 +4,7 @@
 
 RSpec.describe Spree::Bus do
   describe '.publish' do
-    it "doesn't coerce events responding to #to_hash",
-      if: (Gem::Version.new(RUBY_VERSION) < Gem::Version.new("3.0")) && !Spree::Config.use_legacy_events do
+    it "doesn't coerce events responding to #to_hash", if: !Spree::Config.use_legacy_events do
```

Comment:
We can remove this test altogether, as it only was a problem for v2.7.

Reply:
Done 🙏